widget/list.go

```diff
@@ -250,23 +251,39 @@ func (l *List) UnselectAll() {
 	}
 }
 
-func (l *List) visibleItemHeights(itemHeight float32, length int) (visible map[int]float32, offY float32, minRow, maxRow int) {
+func (l *List) visibleItemHeights(itemHeight float32, length int) (visible []float32, offY float32, minRow, maxRow int) {
 	maxRow = length
 	rowOffset := float32(0)
 	isVisible := false
-	visible = make(map[int]float32)
+	visible = []float32{}
 
 	if l.scroller.Size().Height <= 0 {
 		return
 	}
 
+	// theme.Padding is a slow call, so we cache it
```

Comment:
Doesn't this just repeat what the code shows?

Reply:
I am afraid that with time the knowledge of why this is done will be lost.